#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
uint8 aux0
uint8 aux1
uint8 aux2
uint8 aux3
uint8 aux4
uint8 aux5
uint8 aux6
uint8 aux7
uint8 mimic_light
bool left_indicator
bool right_indicator
bool horn
